gbpservice/contrib/nfp/tools/image_builder/elements/root-passwd/README.rst

```diff
@@ -0,0 +1,5 @@
+This element assigns a password to the root account in the image and enables
+password login via ssh.
```

Comment:
Lets remove the root-passwd directory as there is password-less login with ssh private/public key for ubuntu user.